Shared/Realm.Shared/Linq/IRealmCollection.cs

```diff
@@ -110,12 +110,6 @@ internal Move(int from, int to)
     public interface IRealmCollection<out T> : IReadOnlyList<T>, INotifyCollectionChanged, INotifyPropertyChanged
     {
         /// <summary>
-        /// Gets the <see cref="ObjectSchema"/> of the contained objects.
-        /// </summary>
-        /// <value>The ObjectSchema of the contained objects.</value>
-        ObjectSchema ObjectSchema { get; }
```

Comment:
I'd be surprised if anyone was using it, but yes, it's a breaking change. I'm removing it because I'd be surprised if anyone was using it. Do you suggest deprecating it instead?